### DISCUSSION TITLE
Простенькая система учета клиентов с автоматическим формированием пакета документов.

### DISCUSSION THREAD

**Discussion by Kmetyan** [totumonline/totum-ru-issues-and-discussions#12] [Сделанные проекты]
Реализованная на TOTUM  cистема учета клиентов, автоматизация журнала оказанных услуг, формирование документов.

Приветствую! 
В связи с ростом числа клиентов, возникла необходимость в автоматизации процесса формирования комплекта документов (АКТ, счет, инструкция и тд.). Провел много времени для поиска простого и подходящего решения на замену EXEL таблицы учета клиентов, которую вели мои коллеги. В инете есть разные решения, но в основном они требуют достаточного знания и опыта программирования что для меня никак не подходило, я не программист, максимум с чем имел дело это изменение кода php/html/css уже на готовом сайте. 

Итак, моя цель заменить таблицу EXEL на ее WEB аналог.
Решение искал по следующим основным параметрам: 
1. Русская документация; 
2. Готовый проект должен работать на WEB; 
3. При разработке проекта использовать минимум кода.

Поиск привел меня на HABR.COM (https://habr.com/ru/post/511162/), прочел статью, решил, что стоит посмотреть.  Перейдя на сайт TOTUM удивило наличие русской документации с подробным описанием установки TOTUM, через 1 час TUTUM без проблем крутился на VM, далее было несколько дней изучения документации с опытами на TOTUM.

Через 2 недели система работала в боевом режиме на 7 пользователей, все данные (несколько тысяч заказов) из нашего EXEL были успешно перенесены в TOTUM путем импорта из CSV. Так же был разработан функционал по автоматизации формирования документов и вывода их на печать, что значительно сократило время обработки (работая в EXEL документы формировали вручную, на это уходило много времени).
 
Очевидно, но, для себя обнаружили замечательные возможности, предлагаемые технологиями WEB, это возможность работать удаленно!))

Итог: Проект реализован 1 человеком (уверенный пользователь ПК). Кто не имеет достаточного опыта программирования, рекомендую использовать TOTUM для реализации подобных и/или других проектов, порог вхождения довольно низкий. Судя по документации возможности довольно большие в т.ч. для построения CRM/ERP систем.

Выражаю огромную благодарность авторам TUTUM, желаю развития и успехов в дальнейших проектах.

**Comment by ttm-alexeypolunin** [comment]
Спасибо огромное за отзыв! Стараемся сделать документацию как можно более простой и покрывающей множество вариантов использования.
